test/public/defaults.js

```diff
@@ -414,6 +414,28 @@ module.exports.fillInput = async (page, inputSelector, value, events = ['input']
     }, inputSelector, value, events);
 };
 
+/**
+ * Evaluate and return the value content of a given element handler
+ * @param {{evaluate}} elementHandler the puppeteer handler of the element to inspect
+ * @returns {Promise<XPathResult>} the html content
+ */
+const getInnerValue = async (elementHandler) => await elementHandler.evaluate((element) => element.value);
+
+module.exports.getInnerValue = getInnerValue;
```

Comment:
I think it's not useful to export it for now, you only use it internally

Reply:
done